### PR TITLE
Add steps to run minio on cloudron

### PR DESCRIPTION
**Pull request by gramakri**

### CODE REVIEW

**Comment by harshavardhana** [comment]
// cc @NitishT  -  for your review.

**Comment by NitishT** [comment]
Thanks @harshavardhana will take it up today

File: docs/running-minio-in-cloudron.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# How to run Minio in Cloudron [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+[Cloudron](https://cloudron.io) is a platform that makes it easy to run and maintain web apps on your server.
+
+In this recipe, we will learn how to run Minio in Cloudron.
+
+## 1. Prerequisites
+
+* You have Cloudron installed and running, if now follow [install instructions](https://cloudron.io/get.html#selfhost).
```
Comment (NitishT) [inline docs/running-minio-in-cloudron.md]: The line `if now follow` is not clear. Please update.

**Comment by gramakri** [comment]
@NitishT updated, thanks

**Pull request merged**